tests/unit/topology/allocator_binder_linear.hpp

```diff
@@ -24,10 +24,10 @@ struct linear_numa_binder : hpx::compute::host::numa_binding_helper<T>
     explicit linear_numa_binder(std::size_t num_pages)
       : hpx::compute::host::numa_binding_helper<T>()
     {
-        const std::size_t CACHE_LINE_SIZE = hpx::threads::get_cache_line_size();
-        const std::size_t PAGE_SIZE       = hpx::threads::get_memory_page_size();
-        const std::size_t ALIGNMENT       = (std::max)(PAGE_SIZE, CACHE_LINE_SIZE);
-        elements_page_ = (ALIGNMENT / sizeof(T));
+        const std::size_t cache_line_size = hpx::threads::get_cache_line_size();
+        const std::size_t page_size = hpx::threads::get_memory_page_size();
+        const std::size_t alignment = (std::max)(page_size, cache_line_size);
+        elements_page_ = (alignment / sizeof(T));
```

Comment:
While we're at it here... could we change the code to use east-const conventions as well?

Reply:
Ooh, yes. I'll change that. I wish `clang-format` did this for us (it might be coming, but it's not there yet, AFAICT).

Reply:
I think clang-tidy can do this (https://clang.llvm.org/extra/clang-tidy/checks/misc-misplaced-const.html). clang-format is not meant to do contextual changes. Thanks!